docs/getting_started/index.rst

```diff
@@ -57,16 +79,35 @@ ctapipe repo (make sure you put in your own username there):
     $ cd ctapipe
 
 
-You now need to tell Git that this repo where the master CTA version is:
+You now need to tell Git that this repo where the main ctapipe repository is:
 
 .. code-block:: console
 
     $ git remote add upstream https://github.com/cta-observatory/ctapipe.git
 
 If that worked, then you should see a *upstream* target in
-addition to *origin* when typing ``git remote -v``.  Later if you want
-to pull in any changes from the master repo, you just need to type
-``git pull upstream master``.
+addition to *origin* when typing ``git remote -v``.
+
+To get changes from the main repository, run:
+
+.. code-block:: console
+
+   $ git fetch upstream
+
+Then, update a local branch using
+
+.. code-block:: console
+
+   $ git merge upstream/main
```

Comment:
Why go for fetch+merge here, rather than just `git pull upstream main`? Should we have them set the pull policy  to _merge_ for example?

Reply:
I think `git pull upstream main` does not fetch new tags, which is e.g. why @Tobychev had issues with the automatic versioning in his fork.

Reply:
Just verified. `git fetch upstream` fetches new tags, `git merge upstream main` does not.

Reply:
`git pull` has a `--tags` option, which according to the docs is on by default, though testing it it seems not to be the case. Anyhow, I agree that's confusing that `pull`, which should be `fetch + merge` acts slightly differently, so just asking to do it in two steps is probably fine.  

```man
       --no-tags
           By default, tags that point at objects that are downloaded from the remote repository are fetched
           and stored locally. This option disables this automatic tag following. The default behavior for a
           remote may be specified with the remote.<name>.tagOpt setting. See git-config(1).
     -t, --tags
           Fetch all tags from the remote (i.e., fetch remote tags refs/tags/* into local tags with the same
           name), in addition to whatever else would otherwise be fetched. Using this option alone does not
           subject tags to pruning, even if --prune is used (though tags may be pruned anyway if they are also
           the destination of an explicit refspec; see --prune).
```